source/CMakeLists.txt

```diff
@@ -111,10 +123,16 @@ foreach(build ${DEAL_II_BUILD_TYPES})
 
   #
   # Add compile flags, definitions and (public) feature (recorded in
-  # DEAL_II_TARGETS(|_DEBUG|_RELEASE)).
+  # DEAL_II_TARGETS(|_DEBUG|_RELEASE), and in DEAL_II_COMPILER_SUPPORTS_IPO).
   #
   populate_target_properties(${DEAL_II_TARGET_NAME}_${build_lowercase} ${build})
 
+  if(("${build}" MATCHES "RELEASE") AND ("${DEAL_II_COMPILER_SUPPORTS_IPO}" STREQUAL "YES"))
+    set_property(TARGET ${DEAL_II_TARGET_NAME}_${build_lowercase}
+                 PROPERTY INTERPROCEDURAL_OPTIMIZATION TRUE)
+    message(STATUS "Setting LTO properties on ${DEAL_II_TARGET_NAME}_${build_lowercase}")
+  endif()
```

Comment:
And this one should not be necessary any more.

Reply:
You're calling `populate_target_properties(${DEAL_II_TARGET_NAME}_${build_lowercase} ${build})` right above this line. I don't think you need to set the property a second time.